content/docs/rfcs/55/README.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: 55
+title: 55/STATUS-1TO1-CHAT
+name: Status 1-to-1 Chat
+status: stable
+category: Standards Track
+tags: waku-application
+editor: Aaryamann Challani <aaryamann@status.im>
+contributors:
+- Andrea Piana <andreap@status.im>
+- Pedro Pombeiro <pedro@status.im>
+- Corey Petty <corey@status.im>
+- Oskar Thorén <oskar@status.im>
+- Dean Eigenmann <dean@status.im>
+---
+
+# Abstract
+
+This specification describes how the Status 1-to-1 chat protocol is implemented on top of the Waku v2 protocol. 
```

Comment:
Unsure about this, because theoretically, STATUS-1TO1-CHAT could be implemented on either x3dh or noise, which is discussed in the #overview section. Should I narrow down the spec only to include x3dh?

Reply:
As per our discussion out of band, status-1to1-chat will remain generic over any key exchange mechanism